core/src/main/resources/hudson/model/View/newJob.jelly

```diff
@@ -37,15 +37,15 @@ THE SOFTWARE.
           <div class="header">
             <div class="add-item-name">
               <label for="name">${%ItemName.label}</label>
-              <input name="name" id="name" type="text" />
+              <input name="name" id="name" type="text" tabindex="0" />
               <div class="input-help">&#187; ${%ItemName.help}</div>
               <div id="itemname-required" class="input-validation-message input-message-disabled">&#187; ${%ItemName.validation.required}</div>
               <div id="itemname-invalid" class="input-validation-message input-message-disabled"></div>
               <div id="itemtype-required" class="input-validation-message input-message-disabled">&#187; ${%ItemType.validation.required}</div>
             </div>
           </div>
 
-          <div class="categories flat" />
+          <div class="categories flat" role="radiogroup" aria-labelledby="Items"/>
```

Comment:
*Add support information for common UA and AT product combinations* does not means it is not supported in its default web browser. TBD for others UA (*User agents*).